salt/docs/config/nginx.docs-backend.conf

```
@@ -200,6 +200,11 @@ server {
         }
     }
 
+    # Map archive URLs to {major}.{minor}
+    location ~ ^/([a-z-]*/)?(3|3.12|3.13|3.14)/archives/python-(3.12|3.13|3.14)\.\d+((a|b|rc)\d)?-docs(-(html|pdf-a4|pdf-letter|texinfo|text)\.(zip|tar\.bz2)|\.epub)$ {
+        return 301 https://$host/$1$2/archives/python-$3-docs$6;
+    }
```

This might be clearer?: 

cc @JacobCoffee -- I don't know if you have or know of any guidance on best formatting for these redirect rules.
```suggestion
    # Map archive URLs to {major}.{minor}
    location ~ ^/([a-z-]*/)?(3|3.12|3.13|3.14)/archives/python-(3.12|3.13|3.14)\.\d+((a|b|rc)\d)?-docs-(html|pdf-a4|pdf-letter|texinfo|text)\.(zip|tar\.bz2)$ {
        return 301 https://$host/$1$2/archives/python-$3-docs-$6.$7;
    }
    location ~ ^/([a-z-]*/)?(3|3.12|3.13|3.14)/archives/python-(3.12|3.13|3.14)\.\d+((a|b|rc)\d)?-docs\.epub$ {
        return 301 https://$host/$1$2/archives/python-$3-docs.epub;
    }
```

---

given the nature of the capture groups i think there isnt much formatting that could save us! it looks pretty good to me, imo

I *do* think the separation for epub just adds a whole bunch of duplication and could still sit on the original single location block with the rest... but maybe @ewdurbin has more insight